libs/ui-toolkit/src/components/announcement-banner/announcement-banner.tsx

```diff
@@ -0,0 +1,15 @@
+import classnames from 'classnames';
+import type { ReactNode } from 'react';
+
+export interface BannerProps {
+  children?: ReactNode;
+}
+
+export const AnnouncementBanner = ({ children }: BannerProps) => {
```

Comment:
suggestion (non-blocking): How about `forwardRef` it? So it can be manipulated if needed (e.g. closable, hide-able)

Reply:
I'm happy to add functionality if we decide it's needed, but I'd prefer not to build it in before it's necessary. As the vega.xyz version isn't going to use it for now, I'd rather not.